SPECS/openmpi/openmpi.spec

```diff
@@ -0,0 +1,1139 @@
+# Optional name suffix to use...we leave it off when compiling with gcc, but
+# for other compiled versions to install side by side, it will need a
+# suffix in order to keep the names from conflicting.
+#global _cc_name_suffix -gcc
+
+%global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
+%ifarch aarch64 ppc64le x86_64
+%bcond_without ucx
+%else
+%bcond_with ucx
+%endif
+# ARM 32-bit is not supported by rdma
+# https://bugzilla.redhat.com/show_bug.cgi?id=1780584
+%ifarch %{arm}
+%bcond_with rdma
+%else
+%bcond_without rdma
+%endif
+# No more Java on i686
+%ifarch %{java_arches}
+%bcond_without java
+%else
+%bcond_with java
+%endif
+# Private openmpi libraries
+%global __provides_exclude_from %{_libdir}/openmpi/lib/(lib(mca|ompi|open-(pal|rte|trace))|openmpi/).*.so
+%global __requires_exclude lib(mca|ompi|open-(pal|rte|trace)|vt).*
+# Run autogen - needed for some patches
+%bcond_with autogen
+Summary:        Open Message Passing Interface
+Name:           openmpi%{?_cc_name_suffix}
+Version:        4.1.4
+Release:        9%{?dist}
+License:        BSD AND MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://www.open-mpi.org/
+# We can't use %%{name} here because of _cc_name_suffix
+Source0:        https://www.open-mpi.org/software/ompi/v4.1/downloads/openmpi-%{version}.tar.bz2
+Source1:        openmpi.module.in
+Source3:        openmpi.pth.py3
+Source4:        macros.openmpi
+BuildRequires:  gcc-c++
+BuildRequires:  gcc-gfortran
+# Doesn't compile:
+# vt_dyn.cc:958:28: error: 'class BPatch_basicBlockLoop' has no member named 'getLoopHead'
+#                      loop->getLoopHead()->getStartAddress(), loop_stmts );
+#BuildRequires:  dyninst-devel
+BuildRequires:  hwloc-devel
+# Old libevent causes issues
```

Comment:
Let's remove comments, which doesn't mean anything in the context of Mariner. This one looks like an old comment, which may make sense for Fedora devs, but not in our case.
```suggestion
```